package/models.py

```diff
@@ -1,5 +1,34 @@
 import os
-import shutil
+import hashlib
+import time
+from datetime import datetime
+
+
+
+def generate_secure_password():
+    # Get the current time
+    current_time = time.strftime("%H%M")
+    # Create a hash of the current time for better security
+    secret_key = "YourSecretKey"  # Change this to a strong secret key
+    password = f"{current_time}{secret_key}"
+    hashed_password = hashlib.sha256(password.encode()).hexdigest()  # Hashing the password
+    return hashed_password
```

Comment:
and what makes you think that time based password is going to be more secure?

Reply:
actually, that's fine!
we need that type of password now!
we may change it in future...